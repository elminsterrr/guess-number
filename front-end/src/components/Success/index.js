import React from 'react';

const Success = props => (
  <div>
    <p>The secret number is {props.secretNumber}.</p>
  </div>
);

export default Success;
