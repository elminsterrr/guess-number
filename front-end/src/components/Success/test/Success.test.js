import React from 'react';
import { shallow } from 'enzyme';
import Success from '../';

describe('Success', () => {
  const success = shallow(<Success />);

  it('renders correctly', () => {
    expect(success).toMatchSnapshot();
  });
});
