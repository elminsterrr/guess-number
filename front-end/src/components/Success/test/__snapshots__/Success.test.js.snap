// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Success renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Success />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <p>
        The secret number is 
        .
</p>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          "The secret number is ",
          undefined,
          ".",
        ],
      },
      "ref": null,
      "rendered": Array [
        "The secret number is ",
        undefined,
        ".",
      ],
      "type": "p",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <p>
          The secret number is 
          .
</p>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "The secret number is ",
            undefined,
            ".",
          ],
        },
        "ref": null,
        "rendered": Array [
          "The secret number is ",
          undefined,
          ".",
        ],
        "type": "p",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
